lib/Backends/CPU/CPUDeviceManager.h

```diff
@@ -40,9 +41,15 @@ class CPUDeviceManager : public QueueBackedDeviceManager {
   const uint64_t functionCost_{1};
 
 public:
-  CPUDeviceManager(const DeviceConfig &config)
+  explicit CPUDeviceManager(const DeviceConfig &config)
       : QueueBackedDeviceManager(config),
-        maxMemoryBytes_(config_.getDeviceMemory(2000000000)) {}
+        maxMemoryBytes_(config_.getDeviceMemory(2000000000)) {
+    Stats.incrementCounter("accelerators_used.cpu", 1);
```

Comment:
How about `glow.devices_used.xyz`?  I picked `accelerators` so that the string is a bit more descriptive, but the `glow.` prefix should at least help.